desktop/main.js

```diff
@@ -73,6 +75,9 @@ for (let i = 0; i < process.argv.length; i++) {
 let hasUpdate = false;
 let downloadedVersion;
 
+// Current locale being used for translating menu items
+let preferredLocale = BaseLocaleListener.getPreferredLocale();
```

Comment:
We should never import a `Base` module directly – it's meant to be an internal module that's not used by any other module. Other modules should import the platform-specific version of the module. So I think we should just do this here instead:

```suggestion
let preferredLocale = CONST.DEFAULT_LOCALE;
```

Also, I'd recommend adding some more detail to the comment:

```js
// Note that we have to subscribe to this separately and cannot use Localize.translateLocal,
// because the only way code can be shared between the main and renderer processes at runtime is via the context bridge
// So we track preferredLocale separately via ELECTRON_EVENTS.LOCALE_UPDATED
```